cfpb/cfgov-refresh
# Remove URL and template code for FinEd-specific Privacy Act Statement

This now redirects to /privacy/email-sign-privacy-act-statement/

See: https://GHE/CFGOV/platform/issues/2890

## Removals

- `urls.py` entry for `/fin-ed/privacy-act-statement/`
- Template that said URL rendered
- JS that initialized the mobile menu on said template

## Testing

1. Create a redirect from `/fin-ed/privacy-act-statement/` to the page Generic Email Signup Privacy Act Statement
1. Go to http://localhost:8000/fin-ed/privacy-act-statement/ and see that it ignores the redirect and still loads the old template
1. Pull branch
1. Reload http://localhost:8000/fin-ed/privacy-act-statement/ and see that it redirects to `/privacy/email-sign-privacy-act-statement/`

## Checklist
